apps/marketplace/components/SignupForm/SignupForm.js

```diff
@@ -1,371 +1,299 @@
+                        name="line_manager_email"
+                        id="line_manager_email"
+                        htmlFor="line_manager_email"
+                        label="Your manager's email address"
+                        validators={{
+                          required,
+                          validEmail,
+                          governmentEmail
+                        }}
+                        messages={{
+                          required: "You must provide your manager's email address",
+                          validEmail: 'A validly formatted email is required.',
+                          governmentEmail: ' Email should have a government domain.'
+                        }}
+                      />
+                      <PageAlert as="info">
+                        <p>
+                          Remember to let this person know we’ll be sending them an email requesting their
+                          authorisation.
+                        </p>
+                      </PageAlert>
+                    </div>}
+                  {children}
+                  <p>
+                    <small>
+                      By creating an account you confirm your acceptance of our{' '}
+                      <a href="/terms-of-use" target="_blank" rel="external">
+                        Terms of Use
+                      </a>
+                    </small>
+                  </p>
+                  {currentlySending
+                    ? <LoadingButton />
+                    : <input
+                        className="uikit-btn"
+                        type="submit"
+                        value="Create your account"
+                        onClick={onSubmitClicked}
+                      />}
+                </Form>
+              </article>
+            </Layout>}
         </div>
       </div>
-    )
-  }
+    </div>
+  )
 }
 
 SignupForm.propTypes = {
+  csrf_token: PropTypes.string,
   model: PropTypes.string.isRequired,
-  signupForm: PropTypes.object,
-  buyer_url: PropTypes.string,
-  seller_url: PropTypes.string,
+  signupForm: PropTypes.object.isRequired,
   signupSuccess: PropTypes.bool,
-  action: PropTypes.string,
-  csrf_token: PropTypes.string,
-  form: PropTypes.object.isRequired
+  currentlySending: PropTypes.bool.isRequired,
+  isBuyer: PropTypes.bool.isRequired,
+  emailValidators: PropTypes.object.isRequired,
+  emailErrorMessages: PropTypes.object.isRequired,
+  onSubmitClicked: PropTypes.func.isRequired,
+  onSubmitFailed: PropTypes.func.isRequired,
+  handleSubmit: PropTypes.func.isRequired
 }
 
-const mapStateToProps = state => ({
-  ...formProps(state, 'signupForm'),
-  signupSuccess: state.user.signupSuccess,
-  currentlySending: state.app.currentlySending
-})
-
-const mapDispatchToProps = dispatch => ({
-  handleSignupSubmit: model => dispatch(handleSignupSubmit(model))
-})
+SignupForm.defaultProps = {
+  csrf_token: '',
+  signupSuccess: null
```

Comment:
should that be `false`

Reply:
I normally use null for defaults so if it is false I know it's been set that way from an action